heat/core/tests/test_exponential.py

```diff
@@ -38,7 +38,7 @@ def test_exp(self):
 
         # exponential of longs, automatic conversion to intermediate floats
         int64_tensor = ht.arange(elements, dtype=ht.int64)
-        int64_exp = ht.exp(int64_tensor)
+        int64_exp = int64_tensor.exp()
```

Comment:
Why did you not leave it as the module level function? Did you just want to additionally test whether the method call works as well?

Reply:
to test the method call and to increase the coverage on the dndarray file.